### DISCUSSION TITLE
different client have different transaction records

### DISCUSSION THREAD

**Discussion by Firesuiry** [Chia-Network/chia-blockchain#8746] [Q&A]
In my client 1:
![image](https://user-images.githubusercontent.com/30189673/136251214-12110999-5d78-472a-b3fa-19376eb222b0.png)
![image](https://user-images.githubusercontent.com/30189673/136251238-1004037b-40a3-44e8-aca4-adeaf01b8363.png)
In my client 2:
![image](https://user-images.githubusercontent.com/30189673/136251262-7c6f043a-f272-4c3f-b9b0-f8ae76836031.png)
what is the reason of this? how to solve this problems?

**Comment by loppefaaret** [comment]
close down chia on the client with the missing balance, and remove the wallet database found under your users folder `~\.chia\mainnet\wallet\db\blockchain*`
then open up chia again, and resync the wallet from 0

**Reply by Firesuiry** [reply]
ok，I will try it~

**Comment by xiispace** [comment]
transactions are store in the local db,  they are your **local** operation records. so if you use different client, you will sell different results.  you can copy the db files (~\.chia\mainnet\wallet\db\blockchain*) from client 1 to client 2

**Reply by Firesuiry** [reply]
the income transactions are different between different clients?
I have two income transactions and I think they should be the same.
by the way, the two income was sent into different address generated by the client, will it affect this？

**Reply by Firesuiry** [reply]
and  if i lose the db file in my client1, I will lose the XCH sent to my another addresses generated by the clinet?

**Reply by loppefaaret** [reply]
the balance is stored on the blockchain, it should be correctly reflected when you resync the wallet

**Reply by Firesuiry** [reply]
> the balance is stored on the blockchain, it should be correctly reflected when you resync the wallet

Ok I will reply to you after my wallet resynced~  thank you!

**Reply by Firesuiry** [reply]
> the balance is stored on the blockchain, it should be correctly reflected when you resync the wallet

after resynced, the another 0.00999 xch is missing...
![image](https://user-images.githubusercontent.com/30189673/136662491-74c3632d-3835-487c-a3dc-7fe915314e2d.png)
but in the transactions record it exists.

I think maybe it is because i generate some address in my origin client? is my guess true?
and do you know how to solve this problems?

**Comment by loppefaaret** [comment]
https://xchscan.com/address/xch1l3saxzup09vpedegte4w3e6ark5lnlhzuqqveeke3c2jxpgxd8yq295fg4

**Comment by BrandtH22** [comment]
@Firesuiry , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
